docs/static/config-management.asciidoc

```diff
@@ -0,0 +1,73 @@
+[[config-management]]
+== Configuration Management
+
+Logstash provides configuration management features to make it easier for you to
+manage updates to your configuration over time.
+
+[role="xpack"]
+[[logstash-centralized-pipeline-management]]
+=== Centralized Pipeline Management
+
+//REVIEWERS: I'm not sure if centralized pipeline management is the right term to use here, but if we want to have a generic container for lots of types of configuration management, we need to distinguish pipeline management.
+
+NOTE: Centralized pipeline management is an {xpack} feature that requires a
+paid {xpack} license. 
+
+//REVIEWERS: I'm not sure we want to mention actual types of licenses, like Gold, if they might change in the future. Is there a resource that I can point to that outlines the feature support? Is there other language that I should use here to call out the fact that a paid license is required?
+
+The pipeline management feature in {xpack} centralizes the creation and
+management of Logstash configuration pipelines. From within the pipeline
+management UI, you can add, edit, and delete pipeline configurations.
+
+The pipeline configurations are stored in the .`logstash` index in Elasticsearch.
+Any changes that you make to a pipeline definition in the UI are picked up and
+loaded automatically by all Logstash instances registered to use the pipeline.
+The changes are applied immediately; you do not have to restart Logstash to pick
+up the changes, as long as Logtash is already registered to use the pipeline.
+
+To use centralized pipeline management, you must install {xpack} and specify
+the configuration management settings described in
+{logstash-ref}/setup-xpack.html[Setting up {xpack}].
+
+==== Pipeline management UI
+
+To access the pipeline management UI, open {kib} in your browser and go to
+the Management tab. If you've set up configuration management correctly, you'll
+see an area for managing Logstash. Click the *Pipelines* link.
+
+image::static/images/centralized_config.png[]
+
+Here you can add, edit, or delete Logstash pipeline configurations.
+
+To add a new pipeline, click the *Add* button and specify values for the
+following fields:
+
+[horizontal]
+Pipeline ID::
+A name that uniquely identifies the pipeline. You use this ID when you set up
+{logstash-ref}pipeline-management-settings[configuration management] and specify
+a list of pipeline IDs in the `xpack.management.pipeline.id` setting.
+
+Version::
+Metadata for tracking the version of the pipeline configuration. This information
+is for your use. Logtash doesn't currently manage or validate the version information.
+
+Description::
+A description of the pipeline configuration. This information is for your use.
+
+Pipeline::
+The pipeline configuration.
```

Comment:
Not sure I followed this completely. I'll push my edit soon and you can see if I've gotten it right.